### PR TITLE
Update Example usage

### PR DESCRIPTION
**Pull request by AAP9002**
Proposing update of Example usage to match react build folder defaults and Github branch defaults.

As I have not worked with github action pages before, I did not know where to change in the example, to deploy my react app.

I've added comments by these two values to highlight they may need to be changed.

I found in react the default folder was build, and default branch to be master when creating a new git project.

This project works great, and really helped me setup GH pages fast! All the best

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Create `.github/workflow/deploy.yaml` with the following to build on push.
 on:
   push:
     branches:
-      - "main"
+      - "master" # change to the branch you wish to deploy from
```
Comment (mickmcgrath13) [inline README.md:25]: The default branch should be main:
- https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#about-the-default-branch
```suggestion
      - "main" # change to the branch you wish to deploy from
```

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ jobs:
     - id: build-publish
       uses: bitovi/github-actions-react-to-ghp@v1.2.0
       with:
-        path: dist
+        path: build # change to your build folder
```
Comment (mickmcgrath13) [inline README.md:42]: > I found in react the default folder was build, and default branch to be master when creating a new git project.
As this is the readme/docs, it's meant as an example as people might use many different build systems that output in all sorts of different places.

I'm good with this as-is, and I especially love the comment next to the example <3

**Review by mickmcgrath13 — CHANGES_REQUESTED** [review]
Hey @AAP9002 , thank you for the contribution and the kind words about our project!

I reviewed the changes and request that one thing be change (notes in the inline comment).  You should be able to simply commit the suggestion if you agree.

Let us know if you have any questions.

**Comment by AAP9002** [comment]
Hey @mickmcgrath13
That sounds great and thanks again

**Pull request merged**